README.md

```diff
@@ -263,11 +263,26 @@ docs: [
 If you write TeX-style math in your Markdown, such as `$\sum_{i}^{N} x_i$`, it ends up as raw text on the generated pages. To render expressions, we recommend using [KaTeX](https://katex.org/), a JavaScript library that turns expressions into graphics. To load and trigger KaTeX on every documentation page, we can insert the following HTML:
 
 ```html
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.19/dist/katex.min.css" integrity="sha384-beuqjL2bw+6DBM2eOpr5+Xlw+jiH44vMdVQwKxV28xxpoInPHTVmSvvvoPq9RdSh" crossorigin="anonymous">
-<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.19/dist/katex.min.js" integrity="sha384-aaNb715UK1HuP4rjZxyzph+dVss/5Nx3mLImBe9b0EW4vMUkc1Guw4VRyQKBC0eG" crossorigin="anonymous"></script>
-<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.19/dist/contrib/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>
-```
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.css" crossorigin="anonymous">
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex-copytex@latest/dist/katex-copytex.min.css" crossorigin="anonymous"> 
+
+<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.js" crossorigin="anonymous"></script>
+<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/contrib/copy-tex.min.js" crossorigin="anonymous"></script>
+<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/contrib/auto-render.min.js" integrity="sha384-+VBxd3r6XgURycqtZ117nYw44OOcIax56Z4dCRWbxyPt0Koah1uHoK0o4+/RRE05" crossorigin="anonymous"></script>
 
+<script>
```

Comment:
I think we should still use the onload as before. That should be enough!

Reply:
ok, @josevalim , I removed the copytex plugin.  I also added in the integry sha's for katex.min.css and katex.min.js as they were not present, and seem like a good thing to have.  Thanks!

-- Greg

Reply:
I think copytex was fine. :) The pending biggest question is if we should revert to `onload=` as before .

Reply:
ahhh, I see; I misinterpreted your comment, @josevalim !  I've added back in copytex (which admittedly is super handy to have! 😄 ) and switched back to using `onload=`  I had changed it to the current form (using `document.addEventListener("DOMContentLoaded", ...`) just so that it matched up with the other sections in the README (about Mermaid and Vega-Lite), and I was assuming that `onload=` and `document.addEventListener("DOMContentLoaded", ...` were functionally equivalent (but perhaps they're not?).

-- Greg

Reply:
Note that I'm happy to go with either `onload` or `document.addEventListener("DOMContentLoaded"`, whichever you prefer (I thought you meant to go with `onload`, but now that I'm re-reading your comment I'm not 100% sure that's what you meant).  I also pinned the versions on the copytex plugin just to hopefully avoid future issues if things change and break (thinking back to the problems we had a few years ago with phoenix & npm before it switched over to using esbuild).

Reply:
Let’s go with onload cause that’s what Katex recommend on their website. The closer the better :)

Reply:
roger that; the current version in my fork is using the `onload`.  Agreed; the closer with the KaTeX docs the better.  😄

Reply:
Sorry, do we need to pass all arguments? Why can’t it be like before? :)

Reply:
you're correct; I think I had added in the arguments at one point when I was debugging why it wasn't rendering properly, and then erroneously assumed they were necessary since at some point it suddenly started working. I've removed the arguments and it seems to still work fine. So they're gone now; good catch.

Reply:
oops, now I see why I had added in the args for `renderMathInElement`.  It's necessary for the single dollar sign case (i.e., an equation that's inline).  For example, if you have a paragraph where you say $ F = M A $ - this only renders properly if you have the args to `renderMathInElement`.  And it seems with KaTeX that if you add in the arg for $, then you have to also add in the arg for $$.